Amplify/Categories/DataStore/Model/Internal/Schema/ModelField+Association.swift

```diff
@@ -88,7 +88,7 @@ import Foundation
 ///   by host applications. The behavior of this may change without warning.
 public enum ModelAssociation {
     case hasMany(associatedFieldName: String?)
-    case hasOne(associatedFieldName: String?)
+    case hasOne(associatedFieldName: String?, targetName: String?)
```

Comment:
Is this backward compatible, this is a public enum. Might be used by CLI for codegen

Reply:
thanks for catching this! updating with default `nil` parameter